doc/man3/OSSL_sleep.pod

```diff
@@ -0,0 +1,37 @@
+=pod
+
+=head1 NAME
+
+OSSL_sleep - delay execution for a specified number of milliseconds
+
+=head1 SYNOPSIS
+
+ #include <openssl/crypto.h>
+
+ void OSSL_sleep(uint64_t millis);
+
+=head1 DESCRIPTION
+
+OSSL_sleep() is a convenience function to delay execution of the calling
+thread for (at least) I<millis> milliseconds.  The delay is not guaranteed;
+it may be affected by system activity, by the time spent processing the call
+or by system timer granularity.
+
+=head1 RETURN VALUES
+
+OSSL_sleep() does not return any value.
+
+=head1 HISTORY
+
+OSSL_sleep() was added in OpenSSL 3.1.
```

Comment:
3.2

Reply:
... unless we happen to sneak in ossl_sleep() calls in apps/ or test/ for 3.1...  it could come as a "surprise" with any PR we cherry pick...

Reply:
We should assume we don't do that. I can't see why we would do? In any case we'll to have be careful with all commits we cherry-pick to 3.1 and go through an exercise of updating the 3.2->3.1 references accordingly (if any). But that can be dealt with later once we do the cherry-pick.

Reply:
Done